.circleci/config.yml

```diff
@@ -6,9 +6,15 @@ executors:
       # This image isn't used to build Java project, rather to invoke Docker with Gradle.
       # The version of Docker on cimg/openjdk tends to be slightly behind cimg/base.
       - image: cimg/openjdk:14.0.2
+        auth:
+          username: $DOCKERHUB_USERNAME
+          password: $DOCKERHUB_PASSWORD
   helm:
     docker:
       - image: hypertrace/helm-gcs-packager:0.3.0
+        auth:
```

Comment:
Jobs using this executor are missing the context

Reply:
fixed.